agent_application/docker/docker-compose.yml

```diff
@@ -27,12 +27,17 @@ services:
     depends_on:
       - cqrs-postgres-db
   ssi-agent:
-    image: ssi-agent
-    restart: always
+    #image: ssi-agent
+    build:
+      context: ../..
+      dockerfile: ./agent_application/docker/Dockerfile
+    #restart: always
     ports:
       - 3033:3033
     environment:
       AGENT_CONFIG_LOG_FORMAT: json
       AGENT_CONFIG_EVENT_STORE: postgres
-      AGENT_APPLICATION_HOST: ${AGENT_APPLICATION_HOST:?set it please}
+      # Relative paths can't start or end with '/'
```

Comment:
Is this comment still applicable?

Reply:
I will make a fix that it will automatically remove.